### ISSUE TITLE
command line version switch, e.g. -v and --version

### ISSUE THREAD

**Issue by peterjc** [twolfson/restructuredtext-lint#31]
I would like to be able to ask the command line tool what version is installed, e.g.

```
$ rst-lint --version
...
```

**Comment by twolfson** [comment]
This seems like a reasonable request :+1:

**Comment by twolfson** [comment]
Would you be interested in writing a PR for this?

**Comment by peterjc** [comment]
The API changes ought to be simple, but as far as I can see you only define the version number in ``setup.py`` right now.

I would probably do this in ``restructuredtext_lint/__init__.py`` by adding ``__version__ = '0.14.3'`` (this seems to be a common convention for Python libraries), and modify ``setup.py`` to read the version from here (to avoid having to keep two files in sync).

**Comment by twolfson** [comment]
We have added `--version` support in `0.15.0`. Thanks again for the effort on the PR =)

As requested, changes for the version can be found here: https://github.com/twolfson/restructuredtext-lint/compare/0.14.3...0.15.0

**Comment by peterjc** [comment]
Looks good - thanks!

**Comment by peterjc** [comment]
Overnight it occured to me to double check your ``MANIFEST.in`` would include the new ``VERSION`` file, but since you use this wildcard approach it should be fine:

```
recursive-include restructuredtext_lint *
include *
```

**Comment by twolfson** [comment]
It looks like I didn't do enough testing =/ (#33). I have reverted support for now until I get time to dive in again. Thankfully it looks like there is some good advice in that issue as well.

**Comment by twolfson** [comment]
Successfully reproduced error from https://github.com/twolfson/restructuredtext-lint/issues/33#issue-167993908

**Comment by twolfson** [comment]
Here are the commands I used to make sure the issue was resolved:

```bash
# Start in parent directory to avoid loading `VERSION` from `cwd`
# Navigate into directory
cd restructuredtext-lint/

# Compile latest version (same as release process but without `upload`)
python setup.py sdist --formats=gztar,zip

# Navigate to previous directory
cd -

# Install latest version
pip install --upgrade restructuredtext-lint/dist/restructuredtext_lint-0.16.0.tar.gz

# Lint with latest version
rst-lint restructuredtext-lint/README.rst 

# Inline form of commands above
# cd restructuredtext-lint/ && python setup.py sdist --formats=gztar,zip && cd - && pip install --upgrade restructuredtext-lint/dist/restructuredtext_lint-0.16.0.tar.gz && rst-lint restructuredtext-lint/README.rst 
```

**Comment by twolfson** [comment]
Alright, this has been re-released in `0.17.0` with the fix
